e2e-tests/vars.sh

```diff
@@ -10,7 +10,16 @@ export TEMP_DIR=$(mktemp -d)
 export GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 export VERSION=${VERSION:-$(echo "${GIT_BRANCH}" | sed -e 's^/^-^g; s^[.]^-^g;' | tr '[:upper:]' '[:lower:]')}
 
-export IMAGE=${IMAGE:-"perconalab/percona-postgresql-operator:${VERSION}"}
+export IMAGE_BASE=${IMAGE_BASE:-"perconalab/percona-postgresql-operator"}
+export IMAGE=${IMAGE:-"${IMAGE_BASE}:${VERSION}"}
+export PG_VER="${PG_VER:-14}"
+export IMAGE_PGBOUNCER=${IMAGE_PGBOUNCER:-"${IMAGE_BASE}:main-ppg$PG_VER-pgbouncer"}
+export IMAGE_POSTGRESQL=${IMAGE_POSTGRESQL:-"${IMAGE_BASE}:main-ppg$PG_VER-postgres"}
+export IMAGE_BACKREST=${IMAGE_BACKREST:-"${IMAGE_BASE}:main-ppg$PG_VER-pgbackrest"}
+export IMAGE_BACKREST_REPO=${IMAGE_BACKREST_REPO:-"${IMAGE_BASE}:main-ppg$PG_VER-pgbackrest-repo"}
```

Comment:
We do not have separate image for backrest repo

Reply:
done